packages/mdc-floating-label/README.md

```diff
@@ -85,8 +85,8 @@ Mixin | Description
 
 Method Signature | Description
 --- | ---
-`shake(isValid: boolean, isFocused: boolean) => void` | Label animation class is applied unless isValid or isFocused is true.
-`float(value: string, isFocused: boolean, isBadInput: boolean) => void` | Floats label above text field or select.
+`shake(shouldShake: boolean) => void` | Shakes label if shouldShake is true.
```

Comment:
I agree I think we should automatically remove the shake class. As we chatted about offline I will try using animationend event instead of a setTimeout